dumper/channelDataDumper.go

```diff
@@ -74,10 +77,10 @@ func SoftwareChannelTableNames() []string {
 func ProductsTableNames() []string {
 	return []string{
 		// product data tables
-		"suseproducts",             // clean
+		"suseproducts", // clean
 		// FIXME This table needs a special treatement to check if channels exists. Inser to into.. select .. were
 		// shouldn't this table be parte of channel export tables?
-		//"suseproductchannel",       // add only if there are corresponding rows in rhnchannel // clean
+		"suseproductchannel",       // add only if there are corresponding rows in rhnchannel // clean
```

Comment:
This table should be populated in the channel table names to be export instead of being in the products.
Products will be exported first and then the channel. If we are inserting it here, we will not have channels yet.